### ISSUE TITLE
Enum values not generated

### ISSUE THREAD

**Issue by oskrabanek** [Havunen/DotSwashbuckle#7]
After migration from Swashbuckle 5.4.1 to DotSwashbuckle 3.0.6 the enums list of available values is no longer generated in OpenAPI v2.
```
        "parameters": [
          {
            "in": "query",
            "name": "period",
            "description": "period",
            "type": "string",
            "default": "Current",
>>>
            "enum": [
              "Current",
              "Previous"
            ]
<<<
          }
```
![image](https://github.com/Havunen/DotSwashbuckle/assets/6212595/320e649a-77c2-4f40-9a51-c3e72907a204)

**Comment by Havunen** [comment]
This seems to be a bug indeed, looks like enums are not serialized as enum ref in parameters. I will check

**Comment by Havunen** [comment]
fixed in 3.0.7

**Comment by oskrabanek** [comment]
Works like a charm. Thank you.
